src/widgetastic_patternfly/__init__.py

```diff
@@ -1888,6 +1888,74 @@ def items(self):
         return items
 
 
+class Modal(View):
+    """ Patternfly modal widget
+
+        https://www.patternfly.org/pattern-library/widgets/#modal
+    """
+    ROOT = ('.//div[contains(@class, "modal") '
+            'and contains(@class, "fade") and @role="dialog"]')
+
+    def __init__(self, parent, id=None, logger=None):
+        if id:
+            self.id = id
+            self.ROOT = ParametrizedLocator(
```

Comment:
I think this also can be combined with default ROOT but this is rather optional.

Reply:
Thought about the same thing - technically could compose either, and we've done so in some other widgets, but it gets pretty ugly. I think in this case what is here is easily replaced by anything inheriting.

I'm more concerned about `self.id` not getting set unless its passed. I'm going to push a commit to this to move setting of `self.id`